projects/components/src/table/columns/table-edit-columns-modal.component.scss

```diff
@@ -42,3 +29,30 @@
     align-items: center;
   }
 }
+
+.column-item-container {
+  height: 36px;
+  padding: 2px 8px;
+  cursor: move;
+  background: white;
+
+  display: flex;
+  align-items: center;
+
+  .vertical-grab-handle-icon {
+    color: m.$gray-3;
+    align-self: center;
+    cursor: move;
+  }
+
+  .checkbox {
+    margin-left: 14px;
+    cursor: pointer;
+  }
+
+  &:hover {
+    @include m.raised-shadow-hover();
+    border: none;
+    background: m.$gray-1;
+  }
```

Comment:
Should this be `active`, or perhaps a smaller shadow for `hover`, and a larger shadow for `active`?

Reply:
updated and moved to active instead of hover